src/test/runTest.ts

```diff
@@ -1,8 +1,20 @@
 import * as path from "path";
-
+import { spawnSync } from "child_process";
+import { PythonShell } from "python-shell";
 import { runTests } from "vscode-test";
 
+async function checkPyTestIsInstalled() {
+    const response = spawnSync(PythonShell.defaultPythonPath, ["-m", "pytest"]);
```

Comment:
This isn't quite right, the default python may not have pytest installed, but the user may have configured a different python path through the python extension.

Reply:
The tests are run in a isolated environment so I think no users defined pytest is used.